.github/dependabot.yml

```diff
@@ -177,3 +177,11 @@ updates:
   directory: "/resource_detectors"
   schedule:
     interval: weekly
+- package-ecosystem: bundler
```

Comment:
@ericmustin  - Could you double-check the GitHub actions to make sure I've added the new gems to the appropriate places?